tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -145,26 +146,28 @@ Verify Service Is Available In The Explore Page
   Menu.Navigate To Page    Applications    Explore
   Wait For RHODS Dashboard To Load    expected_page=Explore
   Capture Page Screenshot
-  IF    ${split_last} == ${TRUE}
+  ${version_check}=  Is RHODS Version Greater Or Equal Than  1.23.0
+  IF    "${split_last}"=="${TRUE}" and "${version_check}"=="True"
```

Comment:
My understanding is that we shouldn't use these version checks anymore, see https://github.com/red-hat-data-services/ods-ci/pull/711#discussion_r1131822042

Reply:
done

Reply:
Yep, that was news to me as well. Guess we could talk more in the next automation meeting.